etc/shadowing.yaml.dist

```diff
@@ -0,0 +1,37 @@
+# This file is an example file for using DOMjudge as a shadow system.
+# The `import/import-event-feed` command uses the file etc/shadowing.yaml to
+# read configuration options for shadowing another system following the
+# Contest API specification (https://clics.ecs.baylor.edu/index.php?title=Contest_API).
+
+# The root keys are identifiers used to pass to `import/import-event-feed`, e.g.
+# use `import/import-event-feed exampleA` to import the first contest.
+exampleA:
+    # DOMjudge ID of the contest to import into.
+    id: 2
+    # Normally DOMjudge only allows importing when the `data_source`
+    # configuration option is set to 'configuration and live data external'.
+    # Enabling this option allows importing with other `data_source` values.
+    # Optional.
+    # allow-import-as-primary: true
+    # By default, DOMjudge checks if the external ID of the contest in DOMjudge
+    # matches with the external system to prevent importing to the wrong
+    # contest. If you do want to import into such a contest, enable this
+    # option. Optional.
+    # allow-external-id-mismatch: true
+    # URL to import from.
+    feed-url: https://example.com/api/contests/exampleA/event-feed
+    # Username to use for authenticating to the feed. Optional.
+    username: user
+    # Passowrd to use for authenticating to the feed. Optional.
+    password: password
+    # If set, allow self-signed SSL certificates for the URL and submissions.
+    # allow-insecure-ssl: true
+exampleB:
+    id: 3
+    # Instead of setting feed-url, you can also read events from a file on disk.
+    # If this is set, `feed-url`, `username` and `password` fields are ignored.
+    feed-file: /home/user/sample-feed/event-feed.ndjson
+    # The base path to use for relative URL's from the feed. Currently only used
+    # for submission files with relative URL's. This field is required if
+    # `feed-file` is set.
+    base-path: /home/user/sample-feed/files/
```

Comment:
Maybe give an example of where the  script would look for submission files then.

Reply:
Well that depends on whats in the feed right? But I can add something

Reply:
Shouldn't `x/y` be the other way around? But what I was hoping for is a concrete example with a submission source file.

Reply:
So put in an example json?